content/news/xxxx-xx-xx-mixxx-accepted-for-gsoc-2023.md

```diff
@@ -0,0 +1,15 @@
+title: Mixxx accepted for GSoC 2023!
+authors: Daniel Schürmann
+tags: gsoc, gsoc-2023
+status: draft
+comments: no
```

Comment:
Why are we disabling the comments?

Reply:
Because I don't want to encourage possible candidates to comment here.

Reply:
Understandable, I think we should clarify that in the blogpost though. Turning off comments on public posts is usually interpreted as the authors trying to silence criticism IMO. 

How about we add a sentence after the second-to-last paragraph that goes like: "Please understand that we only engage with potential contributors via the respective channels in the gsoc stream on Zulip or via the official gsoc application process"

Reply:
Good idea. Done.